templates/mopro-example-app/android/app/src/main/res/xml/backup_rules.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
Is all of this normally part of Android app or should it be in gitignore?

Reply:
[e865227](https://github.com/oskarth/mopro/pull/103/commits/e865227a4da54ce7ef8019fae49eb01837bf0cc2)